build.sbt

```diff
@@ -112,8 +111,20 @@ lazy val root = (project in file("."))
   .aggregate(scanamo, testkit, alpakka, refined, catsEffect, joda, zio, pekko)
   .settings(
     commonSettings,
-    publishingSettings,
-    noPublishSettings,
+    publish / skip := true,
+    // releaseVersion := ReleaseVersion.fromAggregatedAssessedCompatibilityWithLatestRelease().value,
```

Comment:
This had to be disabled due to v1.0.2 failing and https://github.com/guardian/gha-scala-library-release-workflow/issues/33